docs/docs/cmd/planner/roster/roster-user-plan-list.md

```diff
@@ -0,0 +1,108 @@
+# planner roster user plan list
+
+Lists all Roster plans for a specified user.
```

Comment:
Since all other roster commands are using `Microsoft Planner` in their description, let's do that here also.
Let's not forget to update the description in the command file as well.